compiler/damlc/daml-stdlib-src/DA/Bifunctor.daml

```diff
@@ -7,107 +7,107 @@ module DA.Bifunctor(Bifunctor(..)) where
 
 -- | A bifunctor is a type constructor that takes
 -- two type arguments and is a functor in /both/ arguments. That
--- is, unlike with 'Functor', a type constructor such as 'Either'
--- does not need to be partially applied for a 'Bifunctor'
+-- is, unlike with Functor, a type constructor such as Either
+-- does not need to be partially applied for a Bifunctor
 -- instance, and the methods in this class permit mapping
--- functions over the 'Left' value or the 'Right' value,
+-- functions over the Left value or the Right value,
 -- or both at the same time.
 --
 -- It is a bifunctor where both the first and second
 -- arguments are covariant.
 --
--- You can define a 'Bifunctor' by either defining 'bimap' or by
--- defining both 'first' and 'second'.
+-- You can define a Bifunctor by either defining bimap or by
+-- defining both first and second.
 --
--- If you supply 'bimap', you should ensure that:
+-- If you supply bimap, you should ensure that:
 --
--- ```
--- @'bimap' 'identity' 'identity' ≡ 'identity'@
--- ```
+-- ````
+-- bimap identity identity ≡ identity
+-- ````
 --
--- If you supply 'first' and 'second', ensure:
+-- If you supply first and second, ensure:
 --
--- ```
--- @
--- 'first' 'identity' ≡ 'identity'
--- 'second' 'identity' ≡ 'identity'
--- @
--- ```
+-- ````
+-- 
+-- first identity ≡ identity
+-- second identity ≡ identity
+-- 
+-- ````
 --
 -- If you supply both, you should also ensure:
 --
 -- ````
--- @'bimap' f g ≡ 'first' f '.' 'second' g@
--- ```
+-- bimap f g ≡ first f . second g
+-- ````
 --
 -- By parametricity, these will ensure that:
 --
--- ```
--- @
--- 'bimap'  (f '.' g) (h '.' i) ≡ 'bimap' f h '.' 'bimap' g i
--- 'first'  (f '.' g) ≡ 'first'  f '.' 'first'  g
--- 'second' (f '.' g) ≡ 'second' f '.' 'second' g
--- @
--- ```
+-- ````
+-- 
+-- bimap  (f . g) (h . i) ≡ bimap f h . bimap g i
+-- first  (f . g) ≡ first  f . first  g
+-- second (f . g) ≡ second f . second g
+-- 
+-- ````
 class Bifunctor p where
     {-# MINIMAL bimap | first, second #-}
 
     -- | Map over both arguments at the same time.
     --
     -- ````
-    -- @'bimap' f g ≡ 'first' f '.' 'second' g@
+    -- bimap f g ≡ first f . second g
     -- ````
     --
     -- Examples:
     --
-    -- ```
-    -- >>> bimap not (+1) (True, 3)
+    -- ````
+    -- daml> bimap not (+1) (True, 3)
```

Comment:
This doesn’t trigger in `daml-doctest`.

Reply:
ok, reverting this change.

Reply:
Opened https://github.com/digital-asset/daml/issues/7553 to improve this